xbmc/guilib/GUIFontTTF.h

```diff
@@ -112,8 +133,8 @@ class CGUIFontTTF
   std::string m_strFilename;
 
   // Stuff for pre-rendering for speed
-  inline Character *GetCharacter(character_t letter);
-  bool CacheCharacter(wchar_t letter, uint32_t style, Character *ch);
+  inline Character* GetCharacter(character_t letter, FT_UInt glyph_index);
```

Comment:
I don't think the `inline` specifier is needed as this method should probably just be declared `private` as I don't see it's use outside this class (not 100% sure)

camelCase please for glyph_index